.envrc

```diff
@@ -1,4 +1,4 @@
-source /cvmfs/pilot.eessi-hpc.org/versions/2021.12/init/bash
+source /cvmfs/software.eessi.io/versions/2023.06/init/bash
 module purge
 module load Nextflow
-module load GROMACS/2020.4-foss-2020a-Python-3.8.2
+module load TensorFlow
```

Comment:
TensorFlow is big and has lots of files, should really use GROMACS again once it is there